io.github.dbchoco.Salawat.metainfo.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>io.github.dbchoco.Salawat</id>
+
+    <name>Muezzin</name>
+    <summary>Islamic prayer times application</summary>
+    <url type="homepage">https://github.com/DBChoco/Salawat</url>
+    <url type="bugtracker">https://github.com/DBChoco/Salawat/issues</url>
+    <url type="contact">https://www.reddit.com/r/Muezzin/</url>
+    <url type="translate"> https://discord.gg/cpF9TTstN5</url>
+
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>MIT</project_license>
+    <description>
+        <p>
+            Islamic prayer times (Adhan) that is highly customizable, lightweight and open source.
+        </p>
+        <p>Features:</p>
+        <ul>
+            <li>Many settings</li>
+            <li>Custom background image</li>
+            <li>Custom Adhan</li>
+            <li>Light and dark mode</li>
+            <li>5 different languages</li>
+        </ul>
+        <p>
+            If you encounter a bug, want to add a language, want to contribute or just want to say Salam, join our discord server or subreddit!
+        </p>
+        <p>
+            Jazzakoum Allahu khairan.
+        </p>
+
+        <p>
+            Keywords: Athan, Adhan, Azan, Islam, Muslim, Prayers, times, Muezzin, Salah, Salat, Namaz, Islamic, Hijri.
+        </p>
+    </description>
+    <content_rating type="oars-1.0" />
+    <launchable type="desktop-id">Salawat/Salawat.desktop</launchable>
```

Comment:
this is why it fail to validate. It's wrong.

Reply:
Isn't the error here ?
![Dekstop file](https://i.imgur.com/CtHfHek.png)
The structure of the tar.gz is

- Salawat
=> Salawat.desktop
=> icon.png
=> Salawat (executable)

Shouldn't I just replace the "Icon" field in the desktop file ("io.github.dbchoco.Salawat") by "icon" ?

Reply:
this has nothing to do with the tarball. NOTHING.

Reply:
```suggestion
    <launchable type="desktop-id">io.github.dbchoco.Salawat.desktop</launchable>
```